app/assets/stylesheets/pin-index.css.scss

```diff
@@ -0,0 +1,130 @@
+$primary-background-color: rgba(237, 242, 242, 0.8);
+
+
+#container{
+  background-color: $primary-background-color;
+}
+
+hr{
+  height: 0;
+  max-height: 0;
+  font-size: 1px;
+  line-height: 0;
+  clear: both;
+  color: $primary-background-color;
+  border: none;
+  border-top: 1px solid #aaaaaa;
+  border-bottom: 1px solid #ffffff;
+}
+
+.pin-outer{
+  background-color: white;
+  border: solid $primary-background-color 2px;
+  max-width: 220px;
+  min-height: 300px;
+  background-color: $primary-background-color; 
+}
+
+div.pin{
```

Comment:
Why put html tag in front of the class name? But you don't do it consistently as well